CHANGELOG.md

```diff
@@ -22,7 +22,7 @@
 
 ## Improvements
 
-    ...
+  - PR #113 Handle ucp.BufferRegion objects in auto_device
```

Comment:
@quasiben Wrong release for changelog, I'll move it. 😆

Reply:
:'( -- sorry @kkraus14